docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1997 @@
+an error and closing the subscription.
+```toml
+experimental_subscription_buffer_size = 200
+```
+
+| Value type          | integer   |
+|:--------------------|:----------|
+| **Possible values** | &gt;= 100 |
+
+Higher values will accommodate higher event throughput rates (and will use more memory).
+
+### rpc.experimental_websocket_write_buffer_size
+Experimental parameter to specify the maximum number of RPC responses that can be buffered per WebSocket client.
+```toml
+experimental_websocket_write_buffer_size = 200
+```
+
+| Value type          | integer                                         |
+|:--------------------|:------------------------------------------------|
+| **Possible values** | &gt;= rpc.experimental_subscription_buffer_size |
+
+
+If clients cannot read from the WebSocket endpoint fast enough, they will be disconnected, so increasing this parameter
+may reduce the chances of them being disconnected (but will cause the node to use more memory).
+
+If set lower than `rpc.experimental_subscription_buffer_size`, connections could be dropped unnecessarily. This value
+should ideally be somewhat higher to accommodate non-subscription-related RPC responses.
+
+### rpc.experimental_close_on_slow_client
+Close the WebSocket client in case it cannot read events fast enough. Allows greater predictability in subscription
+behaviour.
+```toml
+experimental_close_on_slow_client = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+The default behaviour for WebSocket clients is to silently drop events, if they cannot read them fast enough. This does
+not cause an error and creates unpredictability.
+Enabling this setting creates a predictable outcome by closing the WebSocket connection in case it cannot read events
+fast enough.
+
+### rpc.timeout_broadcast_tx_commit
+Timeout waiting for a transaction to be committed when using the `/broadcast_tx_commit` RPC endpoint.
+```toml
+timeout_broadcast_tx_commit = "10s"
+```
+
+| Value type          | string (duration)          |
+|:--------------------|:---------------------------|
+| **Possible values** | &gt; `"0s"`; &lt;= `"10s"` |
+
+Using a value larger than `"10s"` will result in increasing the global HTTP write timeout, which applies to all connections
+and endpoints. There is an old developer discussion about this [here](https://github.com/tendermint/tendermint/issues/3435).
+
+> Note: It is generally recommended *not* to use the `broadcast_tx_commit` method in production, and instead prefer `/broadcast_tx_sync`.
+
+### rpc.max_body_bytes
+Maximum size of request body, in bytes.
+```toml
+max_body_bytes = 1000000
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+### rpc.max_header_bytes
+Maximum size of request header, in bytes.
+```toml
+max_header_bytes = 1048576
```

Comment:
Why is this bigger than `max_body_bytes`?

Reply:
that's the same as the default go maximum header size in the `net/http` package. Maybe in the future we should evaluate the max_body_bytes to see if it is still a good value.